server/controllers/cohort.js

```diff
@@ -0,0 +1,41 @@
+const Cohort = require('../models/cohort.model');
+
+const createCohort = (req, resp) => {
+  const { description, name } = req.body;
+
+  const cohort = new Cohort({
+    name,
+    description
+  });
```

Comment:
Shouldn't we save user id here (as adminId and memberId)?

Reply:
on creation we are saving ownerId and adminId